guide/daily-spending-wallet/contacts.md

```diff
@@ -200,22 +194,22 @@ The contacts screen should offer various features for editing and organization.
 
 ### Importing an address
 
-In this scenario, the user has copied a Lightning address to the clipboard and opened the app. The clipboard is automatically scanned, the app identifies the address and offers the user appropriate options. Here, the user adds the address as a new contact.
+This scenario can be initiated by copying a Lightning address to the clipboard, scanning a QR code, or tapping a payment link (see [payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }})). An address is passed into the application and, where it's recognized and appropriate options are shown to the user. In the example below, the user adds the address as a new contact.
 
 {% include image-gallery.html pages = page.imagesImportAddress %}
 
-This flow can also be initiated by scanning a QR code or tapping a payment link. For more, see [Sending bitcoin &middot; Payment entry points]({{ '/guide/daily-spending-wallet/sending/#payment-entry-points' | relative_url }}).
-
 ### Importing an invoice
 
-Here, the user has scanned a payment request and assigns a contact to the payment.
+This sequence is similar to the one above. The difference is that a payment request was passed into the application, which contains different data and also includes a specific user action, and therefore requires different user flows. The one below shows how a user has scanned a payment request and assigns a contact to the payment.
 
-Some payment request formats may include an address that can [receive daily-spending-wallet repeatedly]({{ '/guide/daily-spending-wallet/requesting/#reusable-payment-requests' | relative_url }}). In this case, the address is added to the contact for future use. For [single use payment requests]({{ '/guide/daily-spending-wallet/requesting/#single-use-payment-requests' | relative_url }}), only the payment is linked.
+Some payment request formats may include an address that can [receive repeatedly]({{ '/guide/daily-spending-wallet/requesting/#reusable-payment-requests' | relative_url }}). In this case, the address is added to the contact for future use. For [single use payment requests]({{ '/guide/daily-spending-wallet/requesting/#single-use-payment-requests' | relative_url }}), only the payment is linked.
 
 Invoices may also contain recipient names. These can be used to suggest the name for a new contact to the user. Names can be spoofed, so they should not be automatically assigned without user approval.
```

Comment:
```suggestion
Payment requests may also contain recipient names. These can be used to suggest the name for a new contact to the user. Names can be spoofed, so they should not be automatically assigned without user approval.
```

Regarding my comment above, referring to them as a payment request makes more sense and aligns more with the requesting page. 

Either way we shouldn't be jumping between calling it a simple payment request and an invoice, which is done in this section.